x-pack/plugins/infra/server/services/metrics_explorer_views/metrics_explorer_views_client.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  Logger,
+  SavedObject,
+  SavedObjectsClientContract,
+  SavedObjectsUpdateResponse,
+} from '@kbn/core/server';
+import Boom from '@hapi/boom';
+import {
+  staticMetricsExplorerViewAttributes,
+  staticMetricsExplorerViewId,
+} from '../../../common/metrics_explorer_views';
+import type {
+  CreateMetricsExplorerViewAttributesRequestPayload,
+  MetricsExplorerViewRequestQuery,
+} from '../../../common/http_api/latest';
+import type {
+  MetricsExplorerView,
+  MetricsExplorerViewAttributes,
+} from '../../../common/metrics_explorer_views';
+import { decodeOrThrow } from '../../../common/runtime_types';
+import type { IInfraSources } from '../../lib/sources';
+import { metricsExplorerViewSavedObjectName } from '../../saved_objects/metrics_explorer_view';
+import { metricsExplorerViewSavedObjectRT } from '../../saved_objects/metrics_explorer_view/types';
+import type { IMetricsExplorerViewsClient } from './types';
+
+export class MetricsExplorerViewsClient implements IMetricsExplorerViewsClient {
+  constructor(
+    private readonly logger: Logger,
+    private readonly savedObjectsClient: SavedObjectsClientContract,
+    private readonly infraSources: IInfraSources
+  ) {}
+
+  static STATIC_VIEW_ID = '0';
+
+  public async find(query: MetricsExplorerViewRequestQuery): Promise<MetricsExplorerView[]> {
+    this.logger.debug('Trying to load metrics explorer views ...');
+
+    const sourceId = query.sourceId ?? 'default';
```

Comment:
`default` is repeated in the `get` and `update` functions. We could create a constant for this, wdyt?